app/UI/nav.tsx

```diff
@@ -67,10 +67,8 @@ export class NavBar extends Component<Props, State> {
     styles = {
         container:
             `h-screen w-16 flex flex-col gap-y-4 overflow-y-auto bg-[${PrimaryBackgroundColor}] py-8 items-center`,
-        icons: (active: boolean, fill?: boolean) => (
-            `w-6 h-6 ${fill ? "fill-current" : "stroke-current"} text-[${
-                active ? PrimaryTextColor : SecondaryTextColor
-            }]`
+        icons: (active: boolean) => (
+            `w-6 h-6 stroke-current text-[${active ? PrimaryTextColor : SecondaryTextColor}]`
```

Comment:
It's better to make
```
const SecondaryTextColor = `text-[#XXXXXX]`
```
in the colors file so that the caller code here does not need to prefix `text-`

Reply:
Refactor colors separately later